.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - '2.7'
+install:
+  - npm install -g dredd
+  - pip install -r requirements.pip coveralls coverage
+script:
+  - "coverage run --source='.' ./manage.py test"
+  - ./manage.py migrate
```

Comment:
It would be nice to move this code outside travis.yml to allow local testing. Would it be possible to install tox and move this to tox.ini?